src/sentry/seer/utils.py

```diff
@@ -117,6 +126,35 @@ class SeerSimilarIssueData:
     # TODO: See if we end up needing the hash here
     parent_group_hash: str | None = None
 
+    @classmethod
+    def from_raw(cls, project_id: int, raw_similar_issue_data: RawSeerSimilarIssueData) -> Self:
```

Comment:
can we add a quick docstring here?

Reply:
Good point. I've gotten so lax now that the linter doesn't yell at me to put a docstring on every function! Will do.